src/Components/RouteManager.js

```diff
@@ -16,13 +16,36 @@ export default function RouteManager() {
     // routesList -> To save the array of routes that the user enters
     const [routesList, setRoutesList] = useState([]);
 
+    //jsonObjectList -> saves routesList as Json object
+    const [jsonObjectList, setJsonObjectList] = useState();
+
     // ---- COMPONENT STATE ----
     // routeManagerContent: The content of the RouteManager component.
     //                      If userDecision is "yes" -> RouteTutorial component
     //                      If userDecision is "no" -> RouteEditor component
     //                      If userDecision is "default" -> RouteEditor component behind PopupWindow component
     let routeManagerContent;
 
+    // --------------------------------------------------------
+    // --------------------------------------------------------
+    // ---- HANDELERS ----
+
+    // DESCRIPTION: Converts routeList into a Json object
+    const handleRoutesToJson = () =>{
+        const tempList = routesList.map(element =>({
+            routeName: element.routeName, stepList: element.stepList //creates a temp list without the isChecked proprty from routesList
```

Comment:
I added another property to the route object in my pull request, perhaps we should add it to the comments.
the property name is: `isExpanded`

Reply:
I have added the new property to the comment.